### PR TITLE
Kk/docs update

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**

### CODE REVIEW

File: docs/integrations/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+.. note::
+    `Neptune-client <../python-api/introduction.html>`_ is our official way of logging experiments and notebooks to Neptune (all integrations use it). If you need more control or explicit logging, you can always use it in your projects as well.
+
+My library is not here. What now?
+---------------------------------
+There are two common paths:
+
+#. You can always use `neptune-client <../python-api/introduction.html>`_, our open source Python library for logging all kinds of data and metadata to `experiments <../learn-about-neptune/experiment_tracking.html>`_.
+#. Contact us directly via mail contact@neptune.ai or post |forum| on our forum to discuss what you need and how we can deliver it.
```
Comment (jakubczakon) [inline docs/integrations/introduction.rst:35]: We should mention intercom as well (that little thing in the corner).

File: docs/integrations/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+.. image:: ../_static/images/overview/framework-logos.png
+   :target: ../_static/images/overview/framework-logos.png
+   :alt: selected integrations
+
+Neptune comes with 25+ integrations with Python libraries popular in machine learning, deep learning and reinforcement learning.
+
+How integrations work?
+----------------------
+Integrations are written using `Neptune-client <../python-api/introduction.html>`_ and provide a convenient way to jump-start working with Neptune and particular library. There is no need to integrate your code manually using neptune-client (it's easy though).
```
Comment (jakubczakon) [inline docs/integrations/introduction.rst:12]: I'd go with "and a library that you are using".

File: docs/integrations/introduction.rst (hunk 1 above)
Comment (kamil-kaczmarek) [reply docs/integrations/introduction.rst]: corrected.

File: docs/integrations/introduction.rst (hunk 2 above)
Comment (kamil-kaczmarek) [reply docs/integrations/introduction.rst]: changed.

**Pull request merged**